earth-genome/foodtwin-global-app
# Move globe view to the home page

Contributes to:

- #5 
- #9

Changes:
- Promote globe view from the sandbox to the home page
- Add rewrite rule `/area/:areaId -> /` to avoid rerendering the globe view when the route changes
- Added custom environment variable to enable Stately Inspector
- Moved sandbox index to `/sandbox` route

How to test:
- Visit the home page, click on any country, the browser URL should change without reloading the page
- Visit http://localhost:3000/sandbox, it should display the index of exercise pages
- Set `NEXT_PUBLIC_USE_STATELY_INSPECTOR=true` in `.env`, the Stately Inspector should open in a separate tab

Known issues:
- The home page doesn't use MVT layers, we can add them after we decide which library works best
- If you visit a route like `/area/CA` directly, the country won't appear selected in the map, but it should be straightforward to implement later

@oliverroick this is ready for review. The [rewrite rules](https://nextjs.org/docs/pages/api-reference/next-config-js/rewrites) approach seems the cleanest to make the app work like a SPA, but I'm happy to discuss alternatives.
